app/scripts/modules/core/loadBalancer/loadBalancerServerGroup.directive.js

```diff
@@ -25,6 +25,10 @@ module.exports = angular.module('spinnaker.core.loadBalancer.serverGroup', [
           };
         }
 
+        scope.getRegion = () => {
+          return scope.serverGroup.region || scope.loadBalancer.region;
+        };
```

Comment:
Can this be made a field on the `scope` instead of a method? Do the `loadBalancer` or `serverGroup` ever change?